src/foraging_gui/Foraging.py

```diff
@@ -3303,6 +3307,11 @@ def _NewSession(self):
         self.WarningLabel_2.setText('')
         self._set_metadata_enabled(True)
 
+        self._ConnectBonsai()
```

Comment:
If we lose connection to bonsai, the error should also happen at the stop the camera (self._stop_camera()).  Can we move this before the self._stop_camera()?

Reply:
The error is always being generated from the restart_logging function. So I suspect its due to the user manually closing bonsai, and not a crash. 

It looks like the stop_camera function has some code to check for bonsai connections already